docs/bots-guide.md

```diff
@@ -346,6 +346,62 @@ client.update_message(dict(
 * My bot works only on some streams.
     * Subscribe your bot to other streams, as described [here](#how-to-run-a-bot).
 
+## Flask Server
```

Comment:
It might be helpful to do the following renames:
- "How to run a bot" -> "How to run a single bot manually"
- "Flask Server" -> "How to run bots using the Flask server"